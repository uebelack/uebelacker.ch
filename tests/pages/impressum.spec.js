import React from 'react';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import { IntlProvider } from 'react-intl';
import messages from '../../i18n';
import Impressum from '../../pages/impressum';

const useRouter = jest.spyOn(require('next/router'), 'useRouter');

useRouter.mockImplementation(() => ({
  pathname: '/',
}));

describe('<Impressum/>', () => {
  it('should render', async () => {
    const { container } = render(
      <IntlProvider
        locale="en"
        defaultLocale="en"
        messages={messages.en}
      >
        <Impressum />
      </IntlProvider>,
    );
    expect(container).toMatchSnapshot();
  });
});
