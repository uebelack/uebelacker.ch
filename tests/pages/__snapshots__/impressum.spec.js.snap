// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Impressum/> should render 1`] = `
<div>
  <div
    class="page"
  >
    <header
      class="fixed top-0 w-full"
    >
      <div
        class="md:container px-4 my-4 fade-in flex flex-col md:flex-row"
      >
        <a
          class="flex items-center transition ease-in-out delay-150 flex-grow"
          href="/"
        >
          <img
            alt="Übelacker Solutions logo"
            class="dark:hidden"
            data-nimg="1"
            decoding="async"
            height="40"
            loading="lazy"
            src="/logo.svg"
            style="color: transparent;"
            width="40"
          />
          <img
            alt="Übelacker Solutions logo"
            class="hidden dark:inline-block"
            data-nimg="1"
            decoding="async"
            height="40"
            loading="lazy"
            src="/logo_dark.svg"
            style="color: transparent;"
            width="40"
          />
          <div
            class="text-2xl ml-2 pt-0.5"
          >
            Übelacker Solutions
          </div>
        </a>
        <ul
          class="flex pt-2 pl-1 md:pt-2.5"
        >
          <li
            class="mr-2"
          >
            <a
              href="/#blog"
            >
              Blog
            </a>
          </li>
          <li
            class="mr-2"
          >
            <a
              href="/#apps"
            >
              Apps
            </a>
          </li>
          <li
            class="mr-2"
          >
            <a
              href="/#about"
            >
              About
            </a>
          </li>
          <li
            class="mr-2"
          >
            <a
              href="/#contact"
            >
              Contact
            </a>
          </li>
          <li>
            <a
              href="/en"
            >
              en
            </a>
          </li>
          <li>
            |
          </li>
          <li>
            <a
              href="/de"
            >
              de
            </a>
          </li>
          <li
            class="ml-3"
          >
            <a
              href="https://github.com/uebelack"
              title="Github Profile"
            >
              <svg
                fill="currentColor"
                height="1.5em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 496 512"
                width="1.5em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                />
              </svg>
            </a>
          </li>
          <li
            class="ml-2"
          >
            <a
              href="https://www.linkedin.com/in/daviduebelacker/"
              title="LinkedIn Profile"
            >
              <svg
                fill="currentColor"
                height="1.5em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 448 512"
                width="1.5em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
                />
              </svg>
            </a>
          </li>
        </ul>
      </div>
    </header>
    <div
      class="section"
    >
      <div
        class="prose prose-uebelack dark:prose-uebelackdark md:container px-4"
      >
        <h1>
          Impressum
        </h1>
        <h2>
          Kontaktadresse
        </h2>
        <p>
          Übelacker Solutions GmbH
          <br />
          Bläsiring 28
          <br />
          4057 Basel
          <br />
          Schweiz
        </p>
        <p>
          E-Mail: support@uebelacker.ch
          <br />
          Webseite: uebelacker.ch
        </p>
        <h2>
          Vertretungsberechtigte Person(en)
        </h2>
        <p>
          David Übelacker
        </p>
        <h2>
          Handelsregister-Eintrag
        </h2>
        <h3>
          Eingetragener Firmenname:
        </h3>
        <p>
          Übelacker Solutions GmbH
        </p>
        <h3>
          Unternehmens-Nr (UID):
        </h3>
        <p>
          CHE-175.183.669
        </p>
        <h3>
          Mehrwertsteuer-Nummer
        </h3>
        <p>
          CHE-175.183.669 MWST
        </p>
        <h2>
          Haftungsausschluss
        </h2>
        <p>
          Der Autor übernimmt keinerlei Gewähr hinsichtlich der inhaltlichen Richtigkeit, Genauigkeit, Aktualität, Zuverlässigkeit und Vollständigkeit der Informationen.
        </p>
        <p>
          Haftungsansprüche gegen den Autor wegen Schäden materieller oder immaterieller Art, welche aus dem Zugriff oder der Nutzung bzw. Nichtnutzung der veröffentlichten Informationen, durch Missbrauch der Verbindung oder durch technische Störungen entstanden sind, werden ausgeschlossen.
        </p>
        <p>
          Alle Angebote sind unverbindlich. Der Autor behält es sich ausdrücklich vor, Teile der Seiten oder das gesamte Angebot ohne besondere Ankündigung zu verändern, zu ergänzen, zu löschen oder die Veröffentlichung zeitweise oder endgültig einzustellen.
        </p>
        <h3>
          Haftungsausschluss für Links
        </h3>
        <p>
          Verweise und Links auf Webseiten Dritter liegen ausserhalb unseres Verantwortungsbereichs. Es wird jegliche Verantwortung für solche Webseiten abgelehnt. Der Zugriff und die Nutzung solcher Webseiten erfolgen auf eigene Gefahr des jeweiligen Nutzers.
        </p>
        <h3>
          Urheberrechte
        </h3>
        <p>
          Die Urheber- und alle anderen Rechte an Inhalten, Bildern, Fotos oder anderen Dateien auf dieser Website, gehören ausschliesslich der Firma Übelacker Solutions GmbH oder den speziell genannten Rechteinhabern. Für die Reproduktion jeglicher Elemente ist die schriftliche Zustimmung des Urheberrechtsträgers im Voraus einzuholen.
        </p>
      </div>
    </div>
    <footer
      class="md:container mt-20 mb-20"
    >
      <div
        class="mb-2 px-4 my-4"
      >
        ©
         
        2023
         
        Übelacker Solutions GmbH
        <br />
        <a
          href="/en/imprint"
        >
          Imprint
        </a>
         
        |
         
        <a
          href="/en/privacy"
        >
          Privacy Policy
        </a>
      </div>
    </footer>
  </div>
</div>
`;
